newrelic-integration-e2e/internal/agent/agent.go

```diff
@@ -225,15 +226,20 @@ func (a *agent) Run(scenarioTag string) error {
 		return fmt.Errorf("fail to set %s env: %w", exportersDirEnv, err)
 	}
 
-	return dockercompose.Run(a.dockerComposePath, a.containerName, envVars)
+	// TODO: Distinguish between these two.
+	a.compose.Env(envVars)
+	a.compose.BuildArgs(envVars)
```

Comment:
Perhaps the Dir env vars should be here too.

Reply:
Dir env vars are used in the default docker-compose.yml file to define mount points, therefore they should be on the host :(

Reply:
ho is true that was the reason then i added as host variables.